docs/Tech_Grants_2021.md

```diff
@@ -181,6 +187,7 @@ This ensemble gives the following results on the test set:
 
 Running:
 ```
-python -i nutrition_labels/ensemble_grant_tagger.py --config_path configs/ensemble/2021.04.02.ini
+python nutrition_labels/predict.py --config_path configs/predict/2021.04.02.ini
```

Comment:
Just one thought that occurred to me that it is more intuitive that the predict function will receive an input (say `--data-to-predict-on`). This way it's not tied to the code. I know it's being defined in `grants_data_path` now, but it would be good to give a way to pass new file. This way, we can also more easily put that into an API that receives new data or any way we want to deploy.

Reply:
that sounds good, I could write it so it can do both (if the `data-to-predict-on` arg is passed then use this). 
So are you thinking `python nutrition_labels/predict.py --data-to-predict-on ['grant description 1', 'another grant description 2'] --config_path configs/predict/2021.04.02.ini` (where the config file doesnt need the grants_data_path anymore)?

Reply:
That will do.

Reply:
Or even a path `--data-to-predict-on data.csv`. By the way, the argument doesn't need to be called this way, there might be a better name.

Reply:
hope you don't mind me adding this as [an issue](https://github.com/wellcometrust/nutrition-labels/issues/80) for now - just so I can get the PR closed and some data structure organised